checko/Cargo.toml

```diff
@@ -18,6 +18,7 @@ comfy-table = { workspace = true }
 dioxus.workspace = true
 dioxus-liveview.workspace = true
 dioxus-ssr = "0.4.0"
+dunce = { workspace = true }
```

Comment:
Can we change this to `dunce.workspace = true`?

Reply:
Yes, I've now addressed this in the latest commit 🙂